kubernetes/db/ignite/ignite_alcor.yaml

```diff
@@ -41,16 +39,18 @@ rules:
       - list
       - watch
 ---
-apiVersion: v1
-kind: Namespace
```

Comment:
@DavidLiu506 These two yaml files look exactly the same, except that the order of configurations changes. Is that the case for other yaml files?

Reply:
I order it to make it more clear. ignite_alcor.yaml ignite_alcor_dpm.yaml ignite_alcor_ip.yaml ignite_alcor_mac.yaml these files are same ordered. others is not same. others updated for Kubernetes authentication.

Reply:
Got it. Thanks!